### PR TITLE
Translation strings for the Rythm presence

### PR DESCRIPTION
**Pull request by ACertainCoder**
https://github.com/PreMiD/Presences/pull/3912

### CODE REVIEW

File: src/Presence/rythm.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        "description": "Button for opening the web dashboard."
+    },
+    "rythm.inviteBot": {
+        "message": "Invite Bot",
+        "description": "Button for inviting the bot."
+    },
+    "rythm.djs": {
+        "message": "DJs:",
+        "description": "The amount of users with DJ permissions will be listed after the \":\""
+    },
```
Comment (Bas950) [inline src/Presence/rythm.json:13]:
```suggestion
    "rythm.djs": {
        "message": "DJs: {0}",
        "description": "{0} Will be replaced with the amount of users with DJ permissions."
    },
```
Or if you have the count on another line:
```suggestion
    "rythm.djs": {
        "message": "DJs: {0}{1}",
        "description": "{0} is an enter and is required. {1} Will be replaced with the amount of users with DJ permissions."
    },
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
Please take a look at the suggestion and do it for the other 2 too.

**Comment by Bas950** [comment]
Link your presence pr please

**Comment by ACertainCoder** [comment]
I didn't create a pull request yet because there are a few code changes that I'd have to test first.

- All changes regarding the localization
grafik
- A new Rythm information generation system to prevent the presence from breaking just because they added a new Rythm X to the list.
grafik

Should I open a pull request anyway?

**Pull request merged**